### PR TITLE
[Innawoods] New lost tribal profession

### PR DESCRIPTION
**Pull request by Sathra225**
#### Summary
Mods "Primitive start profession"

#### Purpose of change
Very low-tech start added with reasonable warmth clothing and basic cutting tool and storage. Its a shortcut to skip the 'make basic clothing' step for people wanting to play Innawoods from as close to the bottom as possible but not suffer horribly for a week in-game.

#### Describe the solution
Adds a low-level start with some basic leather clothing wraps, stone chopper, basket and cloak as an alternative to naked if wanting to go with "make everything yourself" but avoiding the whole die to cold and sleep deprivation. Starts with survival 2 and cooking 2 as they seemed thematic for a gatherer.

#### Describe alternatives you've considered
None, been considering adding this for a while.

#### Testing
Made the changes, started a new game and all the gear and skills were as written.

#### Additional context
The leather cloak is a pretty major upgrade and should allow sleeping early on in spring.

### CODE REVIEW

**Comment by Sathra225** [comment]
@Light-Wave for review

File: data/mods/innawood/professions.json [hunk 1]
```diff
@@ -19,5 +19,19 @@
...
+    "type": "profession",
+    "id": "lost_tribal",
+    "name": "Lost Tribal",
+    "description": "You were out gathering for the tribe when the sky broke and the gods called to you, then you collapsed.  When you woke up, you were in an unfamiliar place with new memories crowding your thoughts.",
+    "points": 2,
+    "skills": [ { "level": 2, "name": "survival" }, { "level": 2, "name": "cooking" } ],
+    "proficiencies": [ "prof_fibers", "prof_knapping" ],
+    "items": {
+      "both": {
+        "items": [ "cloak_leather", "footrags_leather", "chestwrap_leather", "loincloth_fur", "stone_chopper", "straw_basket" ]
```
Comment (Light-Wave) [inline data/mods/innawood/professions.json:33]: A wicker backpack could be a more powerful alternative to the straw basket, if you were to feel like the start is underpowered for its point cost. But perhaps that would be a bit too powerful?

Anyways, if you feel it is balanced as it is, I approve this.

**Review by Light-Wave — APPROVED** [review]
A wicker backpack could be a more powerful alternative to

File: data/mods/innawood/professions.json (hunk 1 above)
Comment (Sathra225) [reply data/mods/innawood/professions.json:33]: I did consider that, but the basket also gives impetus to immediate upgrades. Plus its just a nice touch for a tribal gatherer yanked completely unprepared to an alien world/location.

**Pull request merged**